lnwallet/channel.go

```diff
@@ -7585,9 +7581,7 @@ func (lc *LightningChannel) ForceClose() (*LocalForceCloseSummary, error) {
 			"summary: %w", err)
 	}
 
-	// Set the channel state to indicate that the channel is now in a
-	// contested state.
-	lc.status = channelDispute
+	lc.status = channelClosed
```

Comment:
This feels more like a bug fix than pure refactoring and deserves its own commit.

Reply:
Yeah. I can do that. For the record, I think all of this is dead code anyway as I'm fairly certain we never actually hit the error cases that these are for.